### DISCUSSION TITLE
Mac mini 2014

### DISCUSSION THREAD

**Discussion by rlarudqls630** [coder/code-server#4557] [General] [answered]
I am planning to code on iPad.
I personally think that Mac mini 2014 is enough to use it as dedicated code server, also as an home assistant server.
However, my concern is that since it is dual core, it might lag a little. 
What do you guys think about using Mac mini 2014 as a dedicated code server and home assistant server? (Ram will be upgraded to 16gb and SSD will be used.)

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
My opinion: blazing fast. But also you have to _expose_ the service, and I recommend an ethernet connection to your WiFi router so that'd make it easier for data transmission.
